app/src/main/java/ch/deletescape/lawnchair/allapps/AlphabeticalAppsList.java

```diff
@@ -311,7 +311,9 @@ private void onAppsUpdated() {
         // coalesce sections
         Locale curLocale = mLauncher.getResources().getConfiguration().locale;
         TreeMap<String, ArrayList<AppInfo>> sectionMap;
-        boolean localeRequiresSectionSorting = curLocale.equals(Locale.SIMPLIFIED_CHINESE);
+        boolean localeRequiresSectionSorting =
+                Locale.SIMPLIFIED_CHINESE.getLanguage().equals(curLocale.getLanguage())
```

Comment:
This equality should already be granted with the general equality check over Locale itself from what I can see. Could you please show an example of behavior before and after this patch and what the expected behavior is?

Reply:
I think in this case you should compare baseLocale instead, no?

Reply:
getBaseLocale() is not public

Reply:
Any update or better solution?